packages/resolve/src/request-resolver.ts

```diff
@@ -519,12 +519,10 @@ function* matchSubpathPatterns(
       }
       const innerPathStarValue = innerPath.slice(keyPrefix.length, innerPath.length - keySuffix.length);
       const valueStarIdx = valueToMatch.indexOf('*');
-      if (valueStarIdx === -1 || valueToMatch.indexOf('*', valueStarIdx + 1) !== -1) {
+      if (valueStarIdx === -1) {
```

Comment:
@AviVahl 
This if check makes me think maybe there was a reason not to support it?

Reply:
there was no failing case when it was removed 🤷🏻

Reply:
not sure why I added it. probably started with naive supporting only single `*`.